linera-service/src/storage.rs

```diff
@@ -138,24 +130,14 @@ impl StorageConfig {
             } => {
                 let (mut client, table_status) = match use_localstack {
                     true => {
```

Comment:
```suggestion
                let (mut client, table_status) = if use_localstack {
```

And it looks like the `new` call could be deduplicated.